### PR TITLE
feat(kskeleton): component reskin [KHCP-8999]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

[Jira](https://konghq.atlassian.net/browse/KHCP-8999)
[Figma](https://www.figma.com/file/Yze0SWXl5nKjR0rFdilljK/Components?type=design&node-id=2020%3A12479&mode=dev&t=uQGBuUCQfJdk9Uab-1)

`KSkeleton` component reskin

Links:

- [Kongponents sandbox](https://deploy-preview-2082--kongponents-sandbox.netlify.app/#/skeleton)
- [Kongponents docs preview](https://deploy-preview-2082--kongponents.netlify.app/components/skeleton.html)

## PR Checklist

* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [ ] **Tests coverage:** test coverage was added for new features and bug fixes
* [ ] **Docs:** includes a technically accurate README

### CODE REVIEW

File: src/components/KSkeleton/CardSkeleton.vue [hunk 1]
```diff
@@ -3,96 +3,84 @@
...
-          </slot>
-        </div>
+      <div class="skeleton-card-header">
+        <slot name="card-header">
+          <KSkeletonBox width="50" />
+        </slot>
+      </div>
+      <div class="skeleton-card-content">
+        <slot name="card-content">
+          <KSkeletonBox width="100" />
```
[the remark this answers is not in the document]
Comment (portikM) [reply src/components/KSkeleton/CardSkeleton.vue]: Done

File: src/components/KSkeleton/CardSkeleton.vue [hunk 2]
```diff
@@ -3,96 +3,84 @@
...
+          <KSkeletonBox width="100" />
+          <KSkeletonBox width="100" />
+          <KSkeletonBox width="50" />
+        </slot>
+      </div>
+      <div class="skeleton-card-footer">
+        <slot name="card-footer">
+          <KSkeletonBox
```
[the remark this answers is not in the document]
Comment (portikM) [reply src/components/KSkeleton/CardSkeleton.vue:25]: Done

File: src/components/KSkeleton/FormSkeleton.vue [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <div class="skeleton-form-wrapper">
     <div class="skeleton-form-row">
       <KSkeletonBox
```
[the remark this answers is not in the document]
Comment (portikM) [reply src/components/KSkeleton/FormSkeleton.vue]: Done

**Review by DariaYeremina — APPROVED** [review]
LGTM

**Pull request merged**